js/build.js

```diff
@@ -0,0 +1,86 @@
+Fliplet.Widget.instance({
+  name: 'image',
+  displayName: 'Dynamic image',
+  render: {
+    ready: async function() {
+      const image = this;
+      const entryData = image?.parent?.entry?.data || {};
+      const imageInstanceId = image.id;
+      const $imageContainer = $(image.$el);
+
+      image.fields = _.assign(
+        {
+          showIfImageNotFound: 'placeholder',
+          imageColumnName: ''
+        },
+        image.fields
+      );
+
+      let imageOptions = {
+        showPlaceholder: false,
+        url: null,
+        alt: 'Image placeholder',
+        imageColumnName: image.fields.imageColumnName,
+        showIfImageNotFound: image.fields.showIfImageNotFound,
+        placeholderPath: Fliplet.Widget.getAsset(imageInstanceId, 'img/placeholder.jpg')
+      };
+
+      if (Fliplet.Env.get('interface') || Fliplet.Env.get('mode') === 'interact' || !imageOptions.imageColumnName) {
```

Comment:
`Fliplet.Env.get('interface')` Why would `build.js` be run in the `interface` context?